packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.js

```diff
@@ -169,7 +169,7 @@ export default class QuanticRefineToggle extends LightningElement {
   }
 
   get isContentEmpty() {
-    return this.hideSort && !this.someFacetsRendered;
+    return !!this.hideSort && !this.someFacetsRendered;
```

Comment:
hummmm?

Reply:
`hideSort` IS already a boolean no?

Reply:
when it's not provided to the component it's actually `undefined`.

so I'm adding the `!!` to end up with an expression like `false && false` instead of `undefined && false` for example.